BackendApartmentReservation/BackendApartmentReservation/Migrations/20190501130330_AddExternalGroupId.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace BackendApartmentReservation.Migrations
+{
+    public partial class AddExternalGroupId : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+           migrationBuilder.AddColumn<string>(
+                name: "ExternalGroupId",
+                table: "Groups",
+                nullable: false);
```

Comment:
You will most likely not be able to make it not nullable, because there is no default value for that. Therefore, make it nullable